### ISSUE TITLE
Please allow filtering languages in the search result

### ISSUE THREAD

**Issue by mapreri** [aplanas/kmanga#11]
If I search for e.g. "one punch man", the first three results are portuguese-only results.

The user preferences already contains a "preferred languages" selector, I believe the search results should at least consider that value and sort appropriately.

Possibly even better, I would be able to specify a list of languages I can read and the search could return only those (note in this case the search results should contain a "some results were hidden due to language limitation, click to see everything" link)

**Comment by aplanas** [comment]
Right. I implemented something like that, but not in the search. Good point.
